app/serializers/confirmation_serializer.rb

```diff
@@ -0,0 +1,17 @@
+class ConfirmationSerializer
+  
+  def initialize(message)
+    @message = message
+  end
+
+  def info
+    {
+      "data": {
+        "id": nil,
+        "attributes": {
+          "message": @message
+        }
+      }
+    }
+  end
```

Comment:
Again, not sure if this is following the conventions Blinker follows, but I tried my best to adhere to the best practices I've learned at Turing and the JSON API spec.